pages/accountLists/[accountListId]/reports/analysis.page.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { ReactElement } from 'react';
+import Head from 'next/head';
+import { useTranslation } from 'react-i18next';
+import { useRouter } from 'next/router';
+import { Box } from '@material-ui/core';
+import { PartnerGivingAnalysisReportFilters } from '../../../../src/components/Reports/PartnerGivingAnalysisReport/PartnerGivingAnalysisReportFilters';
+import { PartnerGivingAnalysisReportTable } from '../../../../src/components/Reports/PartnerGivingAnalysisReport/PartnerGivingAnalysisReportTable';
+import Loading from '../../../../src/components/Loading';
+
+const PartnerGivingAnalysisReportPage = (): ReactElement => {
+  const { t } = useTranslation();
+  const { query, isReady } = useRouter();
+
+  const { accountListId } = query;
+
+  if (Array.isArray(accountListId)) {
+    throw new Error('accountListId should not be an array');
+  }
```

Comment:
We should make a hook that includes this sooner rather than later. `useAccountListId` or something. I keep thinking about it and have suggested it but never made a ticket. Maybe you should just do it since you're passing it around so much in this PR.